stories/assembly-saas-aap-best-practices.adoc

```diff
@@ -0,0 +1,13 @@
+ifdef::context[:parent-context: {context}]
+
+:context: saas-aap-best-practices
+[id="saas-aap-best-practices"]
+= {PlatformName} best practices
+
+This section covers Ansible Automation Platform product use that has specific content or context for using {PlatformNameShort} as a service.
```

Comment:
I think the attribute for "Ansible Automation Platform" is "{PlatformNameShort}"